src/test/java/unrc/dose/InsertTest.java

```diff
@@ -0,0 +1,33 @@
+package unrc.dose;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+import org.junit.BeforeClass;
+import org.junit.AfterClass;
+
+import org.javalite.activejdbc.Base;
+import org.javalite.activejdbc.DB;
+import spark.Spark;
+
+import unrc.dose.User;
+import org.junit.Test;
+
+
+public class InsertTest {
```

Comment:
This class was used as a way to insert a new user, it should be removed from the Pull Request